source/fundamentals/enterprise-auth.txt

```diff
@@ -267,15 +270,16 @@ mechanism:
 
```

Comment:
cant comment there but on line 264 i think it looks better to italicize everything including the parentheses. Applies in another place as well

Reply:
Updated both